src/SuperGoodRobot/TAPE_FOLLOWING.h

```diff
@@ -0,0 +1,97 @@
+#ifndef TAPE_MENU_H
+#define TAPE_MENU_H
+
+
+//============== DRIVE PARAMS ==============
+
+#include <phys253.h>
+#include <LiquidCrystal.h>
+
+//#include <EEPROMex.h>
+//#include <EEPROMVar.h>
+
+//DRIVE PARAMS
+#define TAPE_SENSOR_LEFT 0
+#define TAPE_SENSOR_RIGHT 1
+
+//QRD DRIVE
+#define QRD_THRESHOLD 200
+
+#define NUM_PRINTSTATES 6
+
+  short saveFlag=500;
+  short loadFlag=500;
+  
+  short error, lerr, recerr = 0;
+  short kp = 20;
+  short kd = 100;
+  short q = 0;
+  short m = 1;
+  short d = 0;
+  short con = 0;
+  short p = 0;
+  short c = 0;
+  short baseSpeed = 150;
+  short printstate=0;
+  //these get changed by the menu/states later
+
+
+void saveEE() {
+  LCD.print("Saving now");
+  EEPROM.write(0,kp);
+  EEPROM.write(1,kd);
+  EEPROM.write(2,baseSpeed);
+}
+
+void loadEE() {
+  LCD.print("Loading now");
+  kp=EEPROM.read(0);
+  kd=EEPROM.read(1);
+  baseSpeed=EEPROM.read(2);
+}
+
+void printLR() {
+  LCD.print("L: "+String(analogRead(TAPE_SENSOR_LEFT)));
+  LCD.print(" R: "+String(analogRead(TAPE_SENSOR_RIGHT)));
+  
+}
+
+void readQRDErrCalc() {
+	//QRD reads
+	short left = analogRead(TAPE_SENSOR_LEFT);
+    short right = analogRead(TAPE_SENSOR_RIGHT);
+    if (left > QRD_THRESHOLD)
```

Comment:
I don't care what the lab code looked like, if multiple lines follow an if/for/while/do etc. statement, make it a code block.  Otherwise, the meaning can become obscure.